src/components/carousel-events/index.js

```diff
@@ -70,7 +70,7 @@ const Events = () => {
                   <SwiperSlide>
                     <div>
                       <div>
-                        <img className="img-foto" src={urlCms + events.image_url} />
+                        <img className="img-photo" src={urlCms + events.image_url} />
```

Comment:
O nome `img-photo` acaba sendo um pouco redundante:

```suggestion
                        <img className="event-img" src={urlCms + events.image_url} />
```